src/platform_impl/windows/window.rs

```diff
@@ -399,6 +399,12 @@ impl Window {
         });
     }
 
+    #[inline]
+    pub fn get_fullscreen(&self) -> Option<RootMonitorHandle> {
+        let window_state = self.window_state.lock().unwrap();
```

Comment:
Looks like windows was added to the list of targets that are using `parking_lot`, which doesn't have `unwrap`.

Reply:
Oh, right. Will correct that, thanks.